src/AutoMapper/ConstructorMap.cs

```diff
@@ -35,19 +35,26 @@ public ConstructorMap(ConstructorInfo ctor, TypeMap typeMap)
 
         public Expression NewExpression(Expression instanceParameter)
         {
-            var parameters = CtorParams.Select(map =>
+            try
```

Comment:
I don't think we need the extra try/catch. Some exceptions are really bugs and need no user friendly messages.

Reply:
Right